src/components/WindowDropDowns/WindowDropDown.js

```diff
@@ -19,7 +19,7 @@ export function WindowDropDown({ items, position = {}, onClick }) {
                   key={item.text}
                   className={`drop-down__row${item.disable ? '--disable' : ''}`}
                   onMouseEnter={() => setOption(item.text)}
-                  onClick={() => onClick(item.text)}
+                  onClick={item.disable ? () => {} : () => onClick(item.text)}
```

Comment:
Just wrap the onClick within the function in the condition, don't pass a different function on runtime